handlers/register_test.go

```diff
@@ -9,7 +9,7 @@ import (
 )
```

Comment:
We can add another test to see if all the end-points are returning nonNil value.